unitary/examples/quantum_chinese_chess/board.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 The Unitary Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unitary.alpha as alpha
+from unitary.examples.quantum_chinese_chess.enums import (
+    SquareState,
+    GameState,
+    Color,
+    Type,
+    Language,
+)
+from unitary.examples.quantum_chinese_chess.piece import Piece
+
+
+class Board:
+    def __init__(self):
+        self.load_fen()
+        self.king_locations = {"e0", "e9"}
+        self.current_state = GameState.CONTINUE
+
+    def load_fen(
```

Comment:
Please add a docstring.  Possibly link to a definition of how FEN format works for quantum chess.

Reply:
Done. Thanks.